ocp_resources/ocs_initialization.py

```diff
@@ -0,0 +1,9 @@
+from ocp_resources.resource import NamespacedResource
```

Comment:
Please follow https://github.com/RedHatQE/openshift-python-wrapper/blob/main/CONTRIBUTING.md for adding a new resource.

Reply:
I have. Is there something specifically wrong?

Reply:
https://github.com/RedHatQE/openshift-python-wrapper/blob/main/CONTRIBUTING.md#adding-a-new-module-resource

Reply:
If you are referring to the missing __init__ function, this resource is not to be created using the API, it's only being added for patching, as stated in the commit message.
I've seen this done in many other resources in the wrapper.
Let me know if it needs to be added regardless.

Reply:
It is needed, we do not add empty resources anymore.

Reply:
added as discussed